### DISCUSSION TITLE
Backrest configuration problem

### DISCUSSION THREAD

**Discussion by hunterbit** [garethgeorge/backrest#558] [General]
Hello everyone,
I performed a few days ago a configuration of both a REPO and a PLAN on a windows server 2019 system and today after logging in I found the Backrest configuration initialized, not even the user account I had created.

What could have happened?
How could I debug and understand why this problem?

Thanks.
![image](https://github.com/user-attachments/assets/0a019f67-9969-4285-83cf-88a7ab17a0a6)

**Comment by garethgeorge** [comment]
Off the top of my head -- 

Backrest keeps logs in %appdata%\backrest\data\processlogs which should be append only and may help a bit here. Those logs wouldn't be reset so they will include log messages dated to the time of your original install IF you find the data directory that backrest install was working with.

The configuration itself lives in %appdata\backrest . Is it possible that your original install is running with / configured in the appdata folder of another windows user?

**Comment by hunterbit** [comment]
My system is multi-user and there may be an access problem because the installation did not go well. I'll do some testing even if in my opinion, since it is a multi-tenant system, everything could be installed in the same path as the application installation, avoiding APPDATA and as you say, it is up to the user to enter a password to prevent access by unauthorized users
